foreign_cc/private/runnable_binary_wrapper.sh

```diff
@@ -19,6 +11,8 @@ source "$(grep -sm1 "^$f " "$0.exe.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/
 { echo>&2 "ERROR: cannot find $f"; exit 1; }; f=; set -e
 # --- end runfiles.bash initialization v2 ---
 
+cd "${RUNFILES_DIR}"
```

Comment:
We may want to catch the case where `RUNFILES_DIR` isn't set or doesn't exist here and show a nice error message (e.g. on Windows, where `RUNFILES_MANIFEST_FILE` would be used instead).